airflow/ui/src/components/TriggerDag/TriggerDAGForm.tsx

```diff
@@ -136,73 +142,74 @@ const TriggerDAGForm: React.FC<TriggerDAGFormProps> = ({
 
   return (
     <>
-      <ErrorAlert error={errorTrigger} />
-      <Accordion.Root collapsible mt={4} size="lg" variant="enclosed">
+      <Accordion.Root collapsible mb={4} mt={4} size="lg" variant="enclosed">
         <Accordion.Item key="advancedOptions" value="advancedOptions">
           <Accordion.ItemTrigger cursor="button">
             Advanced Options
           </Accordion.ItemTrigger>
           <Accordion.ItemContent>
             <Box p={5}>
-              <Text fontSize="md" mb={2}>
-                Data Interval Start Date
-              </Text>
               <Controller
                 control={control}
                 name="dataIntervalStart"
                 render={({ field }) => (
-                  <Input
-                    {...field}
-                    max={dataIntervalEnd || undefined}
-                    onBlur={() => validateDates("dataIntervalStart")}
-                    placeholder="yyyy-mm-ddThh:mm"
-                    size="sm"
-                    type="datetime-local"
-                  />
+                  <Field.Root invalid={Boolean(errors.date)}>
+                    <Field.Label fontSize="md">
+                      Data Interval Start Date
+                    </Field.Label>
+                    <Input
+                      {...field}
+                      max={dataIntervalEnd || undefined}
+                      onBlur={() => validateDates("dataIntervalStart")}
+                      placeholder="yyyy-mm-ddThh:mm"
```

Comment:
I don't know if this is a limitation of the date/time component but if I just use the date picker and leave the time as is (==un-filled) then the component assumes it is "not filled" and the picked date is ignored. Also the validation does not kick-in.

Reply:
Not happening on Chrome - so 'd assume the blame is with the component, not the PR :-D --> :-(

Reply:
Is filling the date and not time, manually checks for validation (Fill only one either start or end)?

Reply:
Yeah, on Firefox when clicking the field it only shows a date picker:
![image](https://github.com/user-attachments/assets/a1fc6fbe-043f-4d48-a876-bd2c61373f97)
...and after selection the date is filled but time is left as "--:--":
![image](https://github.com/user-attachments/assets/48426584-812b-4d5e-8110-aa14572bf6bb)
...and this "half filled" field seems not to be "really" filled until you manually add a time.

Reply:
hmm, should this behaviour be expected?
Now the user has to be very precise with the date-time, they can't leave it empty.
The problem is someone might think they have sent the request and date has been set. But in reality no date has gone in the request.

Reply:
Yes, it seems to be filled after picking a date and having time in `--:--` but actually the field then is not filled and if you submit it is not taken-over.
But this is rather a problem of the `datetime-local` component, irrespective of the PR. And I rememebr @bbovenzi already in some other place made a workaround or fix in 2.10... maybe I find it...

Reply:
Ohh okay!

Reply:
Ah, the discussion I recall... I found it in https://github.com/apache/airflow/pull/39120 - @bbovenzi therefore made a component as workaround... in PR https://github.com/apache/airflow/pull/39261

Maybe worth checking (in a separate PR) if this can be ported over to Airflow 3 code and used as replacement?

Reply:
Then LGTM...